### PR TITLE
Added `kubeConfig` as an argument.

### PR DESCRIPTION
**Pull request by donmstewart**
Added `kubeConfig` as an argument in order that it is possible to add `--kubeconfig=`  to the command line.

### CODE REVIEW

File: pkg/kubernetes/schema_test.go [hunk 1]
```diff
@@ -43,12 +43,14 @@ func TestMixin_ValidatePayload(t *testing.T) {
...
 	}{
 		{"install", "testdata/install-input.yaml", true, ""},
+		{"install-with-kubeconfig", "testdata/install-input-with-kubeconfig.yaml", true, ""},
 		{"upgrade", "testdata/upgrade-input.yaml", true, ""},
 		{"invoke", "testdata/invoke-input.yaml", true, ""},
 		{"uninstall", "testdata/uninstall-input.yaml", true, ""},
 		{"install-bad-wait-flag", "testdata/install-input-bad-wait-flag.yaml", false, "install.0.kubernetes.wait: Invalid type. Expected: boolean, given: string"},
 		{"install-no-manifests", "testdata/install-input-no-manifests.yaml", false, "install.0.kubernetes: manifests is required"},
 		{"install-bad-outputs", "testdata/install-input-bad-outputs.yaml", false, "install.0.kubernetes.outputs.0: resourceType is required\n\t* install.0.kubernetes.outputs.0: resourceName is required\n\t* install.0.kubernetes.outputs.0: jsonPath is required"},
+		{"install-with-kubeconfig", "testdata/install-input-with-kubeconfig.yaml", true, "install.0.kubernetes.outputs.0: resourceType is required\n\t* install.0.kubernetes.outputs.0: resourceName is required\n\t* install.0.kubernetes.outputs.0: jsonPath is required"},
```
Comment (carolynvs) [inline pkg/kubernetes/schema_test.go:53]: I think this test cases was added accidentally? It doesn't appear to be valid.

**Review by carolynvs — CHANGES_REQUESTED** [review]
I think that this is missing changes to https://github.com/getporter/kubernetes-mixin/blob/9aba54a6f99569e4ffe063a01affe47985b84d3c/pkg/kubernetes/execute.go, for the upgrade command. Can you add that in as well?

**Comment by carolynvs** [comment]
@donmstewart The build is broken due to changes unrelated to your PR. I've opened #35 to fix it.

**Comment by carolynvs** [comment]
@donmstewart I merged #35 and updated your PR with the changes so the build is passing now for you.

File: pkg/kubernetes/schema_test.go (hunk 1 above)
Comment (donmstewart) [reply pkg/kubernetes/schema_test.go:41]: The execute command looks to include the install command, the first line of `buildExecuteCommand` calls `buildInstallCommand` so the `kubeConfig` field flows down does it not?

Comment (carolynvs) [reply pkg/kubernetes/schema_test.go]: When I said invalid I was referring to the error that it is expecting to be returned. Shouldn't this test case pass? But instead it has the same expected error message as the "install-bad-outputs" test case above it.

**Comment by carolynvs** [comment]
@donmstewart Sorry about the lag merging this! I didn't see that you pushed more commits after my last comment.

**Review by carolynvs — APPROVED** [review]
This is super useful, thank you!

**Pull request merged**